user-guide/Advanced_Functionality/Databases/Amazon_Keyspaces_Service.md

```diff
@@ -0,0 +1,47 @@
+---
+uid: Amazon_Keyspaces_Service
+---
+
+# Amazon Keyspaces Service
+
+From DataMiner 10.3.0 onwards, it is possible to use the Amazon Keyspaces Service on AWS.
```

Comment:
Can you also add the feature release version?

Reply:
fixed with [dc52c53](https://github.com/SkylineCommunications/dataminer-docs/pull/704/commits/dc52c536c72b6536a609ed8da18678c7743f3f03)